user-guide/Basic_Functionality/Protocols_and_templates/Alarm_templates/Configuring_alarm_templates/Configuring_anomaly_detection_alarms.md

```diff
@@ -2,22 +2,22 @@
 uid: Configuring_anomaly_detection_alarms
 ---
 
-# Configuring anomaly detection alarms for specific parameters
+# Configuring Augmented Operations alarm settings
 
-When you enable alarm monitoring on specific types of anomalies for parameters in an alarm template, an alarm is generated whenever an anomaly of the relevant type is detected for those parameters.
+In an alarm template, you can select to have alarms generated based on Augmented Operations features. You can enable alarm monitoring on specific types of behavioral anomalies for specific parameters, so that an alarm is generated whenever an anomaly of the relevant type is detected for those parameters, and you can configure alarms that should be triggered based on proactive cap detection.
 
 > [!NOTE]
->
-> - Suggestion events are cleared two hours after their creation time or their last update time. You can view them by creating a suggestion event tab in the Alarm Console. See [Adding and removing alarm tabs in the Alarm Console](xref:ChangingTheAlarmConsoleLayout#adding-and-removing-alarm-tabs-in-the-alarm-console).
-> - Anomaly detection is not available for [general parameters](xref:General_parameters)<!--RN 40086-->.
+> Anomaly detection is not available for [general parameters](xref:General_parameters)<!--RN 40086-->.
```

Comment:
These settings (i.e. also proactive cap detection) are not available for general parameters